### PR TITLE
Add notes on specifically downgrading to 1.x and 5.x of Chocolatey products

### PR DESCRIPTION
**Pull request by pauby**
## Description Of Changes
Add notes on downgrading.

## Motivation and Context
See this [internal issue](https://gitlab.com/chocolatey/solutions/support-issues/-/issues/28).

We didn't have any documentation to help with specifically downgrading.

## Testing

**NOTE that YET AGAIN my Docker container is not running.**

* [ ] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

n/a

### CODE REVIEW

File: input/en-us/guides/upgrading-to-chocolatey-v2-v6.md [hunk 1]
```diff
@@ -32,6 +32,19 @@ We have listed the bug fixes, features and improvements below that we feel you s
...
 
+### Downgrading to the Latest 1.x and 5.x Stable Chocolatey Product Versions
+
+If you have issues after upgrading to the 2.x and 6.x versions of Chocolatey products, please see the [support options available](#what-should-you-do-if-you-have-questions).
+
+Before downgrading from 2.x and 6.x to 1.x and 5.x of Chocolatey products, please ensure that you understand the [Support Lifecycle](xref:chocolatey-components-dependencies-and-support-lifecycle). If you still want to downgrade, follow the steps below:
+
+1. If you have Chocolatey GUI and / or Chocolatey GUI Licensed Extension, then uninstall them by running `choco uninstall chocolateygui.extension chocolateygui` from an elevated session.
+1. This steps requires you to downgrade products depending on what you have installed.
```
Comment (imm0rtalsupp0rt) [inline input/en-us/guides/upgrading-to-chocolatey-v2-v6.md:42]: "This step requires..." or "These steps require..."; either would work here in my opinion.

File: input/en-us/guides/upgrading-to-chocolatey-v2-v6.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ When you run the [Chocolatey CLI install command](https://chocolatey.org/install
 
 ### Sonatype Nexus Repository Sources
 
-The Sonatype Nexus Repository Manager has an [issue that can cause it to go into an infinite loop of querying for packages from a NuGet v2 feed](https://issues.sonatype.org/browse/NEXUS-13426) (you will need a login for Sonatypes' Jira system to view the issue). The only affects Sonatype Nexus NuGet v2 feeds as it has been fixed for NuGet v3 feeds.
+The Sonatype Nexus Repository Manager has an [issue that can cause it to go into an infinite loop of querying for packages from a NuGet v2 feed](https://issues.sonatype.org/browse/NEXUS-13426) (you will need a login for the Sonatype Jira system to view the issue). The only affects Sonatype Nexus NuGet v2 feeds as it has been fixed for NuGet v3 feeds.
```
Comment (imm0rtalsupp0rt) [inline input/en-us/guides/upgrading-to-chocolatey-v2-v6.md:50]: > The only affects Sonatype Nexus NuGet v2 feeds as it has been fixed for NuGet v3 feeds.

This only affects Sonatype Nexus NuGet v2 feeds as it has been fixed for NuGet v3 feeds.

File: input/en-us/guides/upgrading-to-chocolatey-v2-v6.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ If you need side-by-side functionality, or cannot uninstall packages installed s
 
 In version 1.0.0 of Chocolatey CLI, we added notices that the `choco list` command will list only local packages, and deprecated the `-l` and it's alias options. See this [GitHub issue for more information](https://github.com/chocolatey/choco/issues/158). We have also removed the `-a` and it's alias options from the `list` command as it no longer made sense to have that option once side-by-side installs were removed.
```
Comment (imm0rtalsupp0rt) [inline input/en-us/guides/upgrading-to-chocolatey-v2-v6.md:141]: > In version 1.0.0 of Chocolatey CLI, we added notices that the `choco list` command will list only local packages, and deprecated the `-l` and _it's_ alias options. See this [GitHub issue for more information](https://github.com/chocolatey/choco/issues/158). We have also removed the `-a` and _it's_ alias options from the `list` command as it no longer made sense to have that option once side-by-side installs were removed.

Please change these to 'its'  as in 'its alias options' with no apostrophe.

**Review by imm0rtalsupp0rt — CHANGES_REQUESTED** [review]

File: input/en-us/guides/upgrading-to-chocolatey-v2-v6.md (hunk 2 above)
Comment (pauby) [reply input/en-us/guides/upgrading-to-chocolatey-v2-v6.md:50]: @imm0rtalsupp0rt Can you come back to me about this?

Comment (imm0rtalsupp0rt) [reply input/en-us/guides/upgrading-to-chocolatey-v2-v6.md:50]: Apologies; I should've been clearer. The sentence says, '_The_ only affects Sonatype Nexus NuGet v2 feeds as it has been fixed for NuGet v3 feeds.' It should say '_This_ only affects . . .'

Comment (pauby) [reply input/en-us/guides/upgrading-to-chocolatey-v2-v6.md]: That's been done.

**Comment by pauby** [comment]
This is ready for final review and merging.

**Comment by imm0rtalsupp0rt** [comment]
LGTM!

**Comment by gep13** [comment]
@pauby thanks for getting this updated!

**Pull request merged**